playbooks/mflux-docker.yml

```diff
@@ -8,7 +8,7 @@
 # to run the playbook for local development, pass -e platform_base=arm64 -e docker_base_image=rockylinux:9 -e docker_tag=dev
 
 # Create a docker container from the local version of the image
-# docker create --name mediaflux-dev --publish 8888:80 --mac-address 02:42:ac:11:00:02 mediaflux-development
+# docker create --name mediaflux --publish 8888:80 --mac-address 02:42:ac:11:00:02 mediaflux-development:{tag_number}
```

Comment:
I see this change, but you did utilize the `docker_tag` on the command line correct?

Reply:
Good catch. I passed the `tag_number` as the docker tag, so i'll just update the line